diff --git a/node_modules/expo-file-system/ios/FilePickingHandler.swift b/node_modules/expo-file-system/ios/FilePickingHandler.swift
index 618e13c..361b008 100644
--- a/node_modules/expo-file-system/ios/FilePickingHandler.swift
+++ b/node_modules/expo-file-system/ios/FilePickingHandler.swift
@@ -1,6 +1,7 @@
 // UIKit is unavailable on macOS, so platform checks are necessary.
 // For macOS support, we should consider using NSOpenPanel: https://developer.apple.com/documentation/appkit/nsopenpanel
-#if os(iOS) || os(tvOS)
+// UIDocumentPickerViewController is unavailable on tvOS
+#if os(iOS)
 import ExpoModulesCore
 import UIKit
 
diff --git a/node_modules/expo-file-system/ios/FilePickingUtils.swift b/node_modules/expo-file-system/ios/FilePickingUtils.swift
index 210d915..6777280 100644
--- a/node_modules/expo-file-system/ios/FilePickingUtils.swift
+++ b/node_modules/expo-file-system/ios/FilePickingUtils.swift
@@ -1,4 +1,5 @@
-#if os(iOS) || os(tvOS)
+// UIDocumentPickerViewController is unavailable on tvOS
+#if os(iOS)
 import ExpoModulesCore
 import MobileCoreServices
 import UIKit
diff --git a/node_modules/expo-file-system/ios/FileSystemModule.swift b/node_modules/expo-file-system/ios/FileSystemModule.swift
index 109d16c..140f3c5 100644
--- a/node_modules/expo-file-system/ios/FileSystemModule.swift
+++ b/node_modules/expo-file-system/ios/FileSystemModule.swift
@@ -4,7 +4,7 @@ import ExpoModulesCore
 
 @available(iOS 14, tvOS 14, *)
 public final class FileSystemModule: Module {
-  #if os(iOS) || os(tvOS)
+  #if os(iOS)
   private lazy var filePickingHandler = FilePickingHandler(module: self)
   #endif
 
@@ -107,7 +107,7 @@ public final class FileSystemModule: Module {
     }
 
     AsyncFunction("pickFileAsync") { (initialUri: URL?, mimeType: String?, promise: Promise) in
-      #if os(iOS) || os(tvOS)
+      #if os(iOS)
       filePickingHandler.presentDocumentPicker(
         picker: createFilePicker(initialUri: initialUri, mimeType: mimeType),
         isDirectory: false,
